### PR TITLE
Update translations for Norwegian Nynorsk (nn) #38756

### LINKED ISSUE #38756: Missing translations for Norwegian Nynorsk (nn)
Hello,

There are some missing translation for Norwegian Nynorsk (nn). These should be added in branch 4.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Form/Resources/translations/validators.nn.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Form/Resources/translations/validators.nn.xlf)
- [src/Symfony/Component/Security/Core/Resources/translations/security.nn.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Security/Core/Resources/translations/security.nn.xlf)
- [src/Symfony/Component/Validator/Resources/translations/validators.nn.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Validator/Resources/translations/validators.nn.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by glye**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #38756
| License       | MIT
| Doc PR        | n/a

Fix https://github.com/symfony/symfony/issues/38756 - Add missing Nynorsk translations, and improve existing ones.

### CODE REVIEW

File: src/Symfony/Component/Security/Core/Resources/translations/security.nn.xlf [hunk 1]
```diff
@@ -36,7 +36,7 @@
             </trans-unit>
             <trans-unit id="10">
                 <source>No session available, it either timed out or cookies are not enabled.</source>
-                <target>Ingen sesjon tilgjengeleg. Sesjonen er anten ikkje lenger gyldig, eller informasjonskapslar er ikke skrudd på i nettlesaren.</target>
+                <target>Ingen sesjon tilgjengeleg. Sesjonen er anten ikkje lenger gyldig, eller informasjonskapslar er ikkje skrudd på i nettlesaren.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Security/Core/Resources/translations/security.nn.xlf:39]: "anten" or "enten"?

**Review by Nyholm — APPROVED** [review]
Awesome. Thank you. 
I've just got a small question. Google translate suggested this. I just want to confirm what you think is the correct word.

File: src/Symfony/Component/Security/Core/Resources/translations/security.nn.xlf (hunk 1 above)
Comment (glye) [reply src/Symfony/Component/Security/Core/Resources/translations/security.nn.xlf:39]: Both are allowed, I verified that. I think "anten" is somewhat preferred. I checked that it's consistently used, no "enten" in these files.

Comment (Nyholm) [reply src/Symfony/Component/Security/Core/Resources/translations/security.nn.xlf:39]: Cool. Thank you.

Comment (glye) [reply src/Symfony/Component/Security/Core/Resources/translations/security.nn.xlf:39]: Official dictionary: https://ordbok.uib.no/perl/ordbok.cgi?OPP=anten&ant_bokmaal=5&ant_nynorsk=5&nynorsk=+&ordbok=nynorsk

**Comment by Nyholm** [comment]
Im happy with this. Let's wait for another maintainer to also give their thumbs up. 

I am happy that you also took the time to improve the existing translations.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @glye.

**Pull request merged**